docs/getting-started/getting-started-rest/03-handling-errors.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Handling Errors
+---
+
+# Handling Errors in Your REST API
+
+## Introduction
+
+In this section, we'll focus on handling errors in your REST API. We've already introduced defining error models in the previous sections, and now we'll expand on that topic. We'll define additional error models, create custom response models for error handling, and demonstrate how to use union types for different response scenarios.
+
+## Defining Error Models
+
+Error models are used to represent different types of errors that your API might return. Let's start by defining some common error models.
+
+### Example: Defining Common Error Models
+
+We've already defined models to represent validation errors and not found errors. We'll now add a model for internal server errors:
+
+```tsp tryit="{"emit": ["@typespec/openapi3"]}"
+import "@typespec/http";
+import "@typespec/rest";
```

Comment:
So looking at those change I don't think the rest library is actually used at all, would it better to keep that guide as http

Reply:
Calling it an HTTP tutorial is incorrect, since HTTP is the protocol. We may not use the rest library but we're defining a REST API.  
I do see your point though about it not using the rest library, I'll remove those references from the tutorial.